packages/tokens/dist/products/css/tokens.css

```diff
@@ -3,6 +3,10 @@
  */
 
 :root {
+  --token-border-radius-x-small: 3px;
+  --token-border-radius-small: 5px;
```

Comment:
[issue/blocker]: comment for @jorytindall and @hashicorp/hds-design in general: this is the first case of t-shirt naming for our design tokens; what if we need to add a `4px` border radius in the future? what would be the name? 

should we use a different naming convention? maybe
```
--token-border-radius-300: 3px;
--token-border-radius-500: 5px;
--token-border-radius-600: 6px;
--token-border-radius-800: 8px;
```
or maybe simply:
```
```
--token-border-radius-3: 3px;
--token-border-radius-5: 5px;
--token-border-radius-6: 6px;
--token-border-radius-8: 8px;
```

Reply:
Let me surface this to the broader team for discussion, it's a valid point. We discussed initially what to do here and I can't quite remember what the reasoning was.

Reply:
Related Slack discussion thread: https://hashicorp.slack.com/archives/C025N5V4PFZ/p1734029185471359